docs/index.rst

```diff
@@ -356,6 +356,28 @@ An example that combines a dbname with a query string::
 
    memory://storagename?connection_cache_size=100&database_name=fleeb
 
+
+``demo:`` URI scheme
+~~~~~~~~~~~~~~~~~~~~
+
+The ``demo:`` URI scheme can be passed as ``zodbconn.uri`` to create a
+DemoStorage database factory. DemoStorage provides an overlay combining base
+and changes storages. The URI scheme contains two parts - base and δ for changes::
```

Comment:
Per [discussion](https://github.com/Pylons/zodburi/pull/29#discussion_r593978001).

```suggestion
and δ ("delta", or in other words, "changes") storage.
The URI scheme contains two parts, base and δ::
```

Reply:
Thanks @stevepiercy. I've reworked the text according to your suggestion with the minor difference that at the end it is storage**s** because we are talking about both base and δ:

```
The ``demo:`` URI scheme can be passed as ``zodbconn.uri`` to create a
DemoStorage database factory. DemoStorage provides an overlay combining base
and δ ("delta", or in other words, "changes") storages.
The URI scheme contains two parts, base and δ::

    demo:(base_uri)/(δ_uri)
```

Hope it is ok.

Reply:
Yes, this is good. Thank you!

Reply:
Thanks.